apps/nextjs-website/src/lib/types/webinar.ts

```diff
@@ -0,0 +1,10 @@
+import { Speaker } from '@/lib/types/speaker';
+
+export type Webinar = {
+  readonly description: string;
+  readonly speakers?: readonly Speaker[];
+  readonly startDateTime?: Date;
```

Comment:
Also the startDateTime shouldn't be required?

Reply:
* for the same reason of the undefined speakers